srcDoc/tutorials/performance_validation.rst

```diff
@@ -0,0 +1,635 @@
+
+    .. code::
+
+        git clone -b release/3.0 https://github.com/rticommunity/rtiperftest.git
+
+   This command will download the *Github* repository in a folder named
+   ``rtiperftest`` and move to the ``release/3.0`` branch.
+   If you don't include the ``-b release/3.0``, you will clone the ``master`` branch
+   of the product.
+
+-  You can download a `zip` file containing the **Perftest** source files from
+   the **Github** page:
+   `github.com/rticommunity/rtiperftest <https://github.com/rticommunity/rtiperftest>`__.
+   Once the zip file is downloaded you will need to extract its content,
+   this will create a folder named ``rtiperftest``.
+
+-  You can download a `zip/tar.gz` file containing the **Perftest** executable statically
+   compiled for some of the most common platforms from the **Github** release page:
+   `https://github.com/rticommunity/rtiperftest/releases <https://github.com/rticommunity/rtiperftest/releases>`__
+   in the "Binaries" section. Once the zip file is downloaded you will need to extract its content, this will
+   create a folder with the binaries for your architecture.
+
+All this information is covered in the `download <https://github.com/rticommunity/rtiperftest/blob/3.0/srcDoc/download.rst>`__
+section of the **Perftest** documentation.
+
+Compile against RTI Connext DDS Professional 6.0.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you already got the compiled binaries, you can skip this step. Else, you will need to compile the
+binaries. This process is covered in the `compilation <https://github.com/rticommunity/rtiperftest/blob/3.0/srcDoc/compilation.rst>`__
+section of the **Perftest** documentation, so we will just summarize.
+
+We want to build **Perftest** for the *Raspberry Pi* target libraries, this architecture is `armv6vfphLinux3.xgcc4.7.2`, even
+though you should be able to compile everything in a raspberry pi, we are going to cross-compile this architecture.
+This process should be really simple with *Perftest*, since we will just need
+to have in the `$PATH` environment variable the path to the compiler and linker for the given architecture.
+
+Therefore the command we will need to execute should look like this:
+
+    .. code::
+
+        export PATH=<Path to the compiler and linker for armv6vfphLinux3.xgcc4.7.2>:$PATH
+        ./build.sh --platform armv6vfphLinux3.xgcc4.7.2 --nddshome <Path to your nddshome> --cpp-build
+
+Alternatively, you can just point to the compiler and linker using the ``--compiler`` and ``--linker``
+command line options. As you can see we also specified the ``--cpp-build`` option, this is because we
+are going to use only that executable to test with.
+
+After executing this, you should have a statically linked binary in `./bin/armv6vfphLinux3.xgcc4.7.2/release`,
+this is all you should need for your testing.
+
+Compile against RTI Connext DDS Micro 3.0.0
```

Comment:
You can compile using the same directory, although the compilation will need to be done in 2 passes (since once would be for pro and the other one for micro). The resulting executables will not overwrite themselves, but live in the same directory with different names.

I will add this information as a note.

Reply:
**Note:** Although you will need to call the build script two times for compiling for **RTI Connext DDS Proffesional**
and **Micro**, you don't need to use two different directories, since the executables will be stored
with different names. It is also worth mentioning that cross-testing (Using a **Perftest** Publisher from **Professional**
and a Subscriber from **Micro** or viceversa) is supported.